.gitignore

```diff
@@ -21,3 +21,8 @@ venv
 .github/env
 requirements.txt
 actions-runner/
+stuff/
```

Comment:
Is this an example or something left in automatically?

Reply:
No, I have `stuff` and `foo.*` ignored globally on my machine. Using the new devcontainer locally mounts the entire working dir which doesn't have my global ignore. Therefore a few hundred files popped up to be commited. So I had to ignore these in the projects gitignore.